src/java/org/apache/cassandra/config/Config.java

```diff
@@ -349,8 +349,15 @@
     public volatile int tombstone_warn_threshold = 1000;
     public volatile int tombstone_failure_threshold = 100000;
 
+    // coordiantor reads
     public volatile long client_large_read_warn_threshold_kb = 0;
     public volatile long client_large_read_abort_threshold_kb = 0;
```

Comment:
It looks like `client_large_read_warn_threshold_kb` and `client_large_read_abort_threshold_kb` are only in trunk. Perhaps we could just prefix them with `coordinator_` here and be rid of the comments?